packages/flutter/test/foundation/error_reporting_test.dart

```diff
@@ -45,13 +45,19 @@ Future<void> main() async {
 
   final StackTrace sampleStack = await getSampleStack();
 
-  test('Error reporting - pretest', () async {
+  setUp(() async {
```

Comment:
Because we need this before any individual test. When we eventually randomize our tests this will be important.